internal/feedback-tool/src/main/java/org/wso2/security/tool/adapter/ExcelInputAdapter.java

```diff
@@ -0,0 +1,106 @@
+    private String logInfo;
+
+    /**
+     * Converts the data in the files with .xlsx extension to the JSON format.
+     * A workbook is created from the the excel file (.xlsx) and while iterating through the sheets in the workbook;
+     * the data is read and  set in to a JSONObject. The JSONObject returned by the method contains an array of
+     * row objects corresponding to each row in the workbook. A row object contains values of each cell in a given row,
+     * with key values starting from letter 'A'.
+     *
+     * @param dataFilePath The path where the data file uploaded is saved.
+     * @return returns the JSON object that contains all the data in the .xlsx file.
+     * @throws FeedbackToolException If the .xlsx file is not found in the given path or due to an error in
+     *                               parsing the data in the data file.
+     */
+    @Override
+    public JSONObject convert(String dataFilePath) throws FeedbackToolException {
+
+        // JSONObject to hold the array of row objects
+        JSONObject dataJSONObject = new JSONObject();
+        try {
+            Workbook workbook = WorkbookFactory.create(new File(dataFilePath));
+            logInfo = "Workbook has " + workbook.getNumberOfSheets() + " sheets";
+            log.info(logInfo);
+
+            Iterator<Sheet> sheetIterator = workbook.sheetIterator();
+
+            // JSONArray to hold all the row objects
+            JSONArray rowsJSONArray = new JSONArray();
+            while (sheetIterator.hasNext()) {
+                Sheet sheet = sheetIterator.next();
+                logInfo = "Sheet: " + sheet.getSheetName() + " has " + sheet.getNumMergedRegions()
+                        + " merged regions";
+                log.info(logInfo);
+
+                DataFormatter dataFormatter = new DataFormatter();
+
+                logInfo = "Iterating over Rows and Columns using for-each loop";
+                log.info(logInfo);
+                for (Row row : sheet) {
+
+                    // JSONObject to hold the data in the cells of a given row
+                    JSONObject rowJSONObject = new JSONObject();
+
+                    char keyLetter = 'A';
+                    for (Cell cell : row) {
+                        String cellValue = dataFormatter.formatCellValue(cell);
+                        rowJSONObject.put(keyLetter, cellValue);
+                        ++keyLetter;
+                    }
+                    rowsJSONArray.add(rowJSONObject);
+                }
+            }
+            dataJSONObject.put(Constants.JSON_DATA_OBJECT, rowsJSONArray);
+        } catch (InvalidFormatException e) {
+            throw new FeedbackToolException("Error in parsing the data file uploaded" + e);
+        } catch (IOException e) {
+            throw new FeedbackToolException("Data file was not found in the specified location" + e);
+        }
+        return dataJSONObject;
+    }
+}
```

Comment:
As per check-style, leave a new line at end of the class. Fix this in each class.